machine1/dbus_test.go

```diff
@@ -181,4 +184,14 @@ func TestImages(t *testing.T) {
 	if len(listImages) < 1 {
 		t.Fatalf("did not find any image")
 	}
+
+	cloneErr := conn.CloneImage(imageName, newImageName, false)
+	if cloneErr != nil {
+		t.Fatal(cloneErr)
+	}
+	defer os.Remove(newImagePath)
```

Comment:
I think you may want to `RemoveImage` here, not removing by FS path.

Reply:
The logic is borrowed from old tests, maybe we should refactor them all?

Reply:
The old test is creating the file directly, while here it happens implicitly (by machined). That wasn't clear to me first at first.

If you have time, I'd be happy to see this test doing the following before exiting:
 * doing `defer os.Remove()` on `imagePath` and `newImagePath`
 * calling `RemoveImage` on both `imageName` and `newImageName`
 * checking that  `imagePath` and `newImagePath` do not exist anymore

Reply:
OK, I will refactor the logic for test.